javascriptv3/example_code/personalize/src/personalize_createEventTracker.js

```diff
@@ -0,0 +1,46 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createEventTracker.js demonstrates how to create an event tracker with Amazon Personalize. You 
+use an event tracker when you record events with the PutEvents operation.
+See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateEventTracker.html.
```

Comment:
For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateEventTracker.html.